ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -1836,6 +1836,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity
+- as: gcp-ipi-proxy-oidc-mini-perm-filestore-csi-arm-f28
```

Comment:
We can directly replace `gcp-ipi-proxy-oidc-mini-perm-arm-f28` with this new one.